web/src/app/wizard/WizardForm.jsx

```diff
@@ -188,6 +188,10 @@ export default function WizardForm(props) {
                 label: 'Email',
                 value: 'email',
               },
+              {
+                label: 'Universal',
+                value: 'universal',
+              },
```

Comment:
I'd say just leave it out from this until/unless we update the wizard to use the API/types hook

Reply:
yeah I think all of the UI should be using the context to grab this list, since it already exists

Reply:
https://github.com/target/goalert/pull/3837